velox/common/memory/MappedMemory.h

```diff
@@ -207,12 +281,25 @@ class MappedMemory {
     return nullptr;
   }
 
+  virtual std::string toString() const;
+
+ protected:
+  MachinePageCount allocationSize(
```

Comment:
- Would you document this method?
- This method would be easier to read and use if it returned a struct with sizeIndices, sizeCounts, numSizes and MachinePageCount.

Reply:
Repeating earlier comment:

----
* Would you document this method?
* This method would be easier to read and use if it returned a struct with sizeIndices, sizeCounts, numSizes and MachinePageCount.